src/frontend/screens/Settings/components/EnableDXVKFpsLimit.tsx

```diff
@@ -36,13 +36,16 @@ const EnableDXVKFpsLimit = () => {
           htmlId="enableDXVKFpsLimit"
           value={enableDXVKFpsLimit || false}
           handleChange={() => setDXVKFpsLimit(!enableDXVKFpsLimit)}
-          title={t('setting.dxvkfpslimit', 'Limit FPS (DX9, 10 and 11)')}
+          title={t('setting.fpslimit', 'Limit FPS')}
         />
 
         <FontAwesomeIcon
           className="helpIcon"
           icon={faCircleInfo}
-          title={t('help.dxvkfpslimit', 'Sets a frame rate cap for DXVK games')}
+          title={t(
+            'help.fpslimit',
+            'Sets a frame rate cap for DXVK/VKD3D games'
```

Comment:
I realize you're just adapting the original wording, but I'd change this to say "DirectX games" (or "DX9-12 games" if you want to be really accurate)